kubernetes-deployment/massdriver.yaml

```diff
@@ -46,50 +67,62 @@ params:
           type: integer
           default: 90
     ingress:
+      title: Ingress
+      description: Configure the application for public
```

Comment:
```suggestion
      description: Configure the application for public internet accessibility
```

Reply:
doing this locally so I dont have to pull from remote.

Reply:
did this locally.